package.json

```diff
@@ -11,11 +11,12 @@
   "license": "MIT",
   "repository": "koajs/trie-router",
   "dependencies": {
-    "methods": "1",
+    "errors": "^0.3.0",
```

Comment:
oh ok... but do we want to create a custom error w/ custom code or a standard http error on malformed URL? like we could throw a 404 from http-errors and have users of this module catch that but we could also through a completely different error that's distinguishable... like `MalformedUrl` error?